library/src/scala/quoted/Quotes.scala

```diff
@@ -3437,6 +3472,9 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
     trait SymbolMethods {
       extension (self: Symbol)
 
+        /** TypeRepr of the definitions of this symbol */
+        def info: TypeRepr
+
```

Comment:
If we expose `info` on symbols, logically we need to expose `Type.asSeenFrom` as well to rebase the prefix.

Reply:
Added

Reply:
Maybe add a document here saying 

> Warning: you may need to perform `asSeenFrom` in case `Symbol.info` contains parameterized types, such as class type parameter, types depend on `this`, etc.
>
> For example, if the owner of the current symbol is a class, using the following code to rebase the type:
>
>     symbol.info.asSeenFrom(prefix, symbol.owner)

What about a method like `symbol.infoAsSeenFrom(prefix)`?